### ISSUE TITLE
Export to Collada does not work in Blender 4.3

### ISSUE THREAD

**Issue by aiaustin** [aiaustin/Onigiri#2]
Export to Collada does not work in Blender 4.3

**Comment by aiaustin** [comment]
```
rigs::make_complete : warning - bone matching sequence may damage devkits
Traceback (most recent call last):
  File "C:\Users\bat\AppData\Roaming\Blender Foundation\Blender\4.3\scripts\addons\Onigiri\__init__.py", line 17420, in execute
    result = rigutils.make_complete(
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\bat\AppData\Roaming\Blender Foundation\Blender\4.3\scripts\addons\Onigiri\rigutils.py", line 3875, in make_complete
    set_bone_groups(armObj)
  File "C:\Users\bat\AppData\Roaming\Blender Foundation\Blender\4.3\scripts\addons\Onigiri\rigutils.py", line 3592, in set_bone_groups
    bpy.ops.pose.group_add()
  File "D:\Programs\Blender Foundation\Blender 4.3\4.3\scripts\modules\bpy\ops.py", line 109, in __call__
    ret = _op_call(self.idname_py(), kw)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: Calling operator "bpy.ops.pose.group_add" error, could not be found
```

**Comment by aiaustin** [comment]
Fixed in Onigiri 4.0.1.0 by @zaher
